docs/modules/ROOT/pages/index.adoc

```diff
@@ -290,6 +290,9 @@ When you write stages, you have access to both global variables (defined without
 
 | ciSkip
 | Whether the build should be skipped, based on the Git commit message.
+
+| dockerDir
+| The docker directory to use when building the image in openshift
```

Comment:
Maybe this could be "The Docker context directory to use when building the image in OpenShift. Defaults to `docker`."?

Reply:
done.